cmd/up/up.go

```diff
@@ -904,11 +916,66 @@ func (up *upContext) shutdown() {
 		up.Forwarder.Stop()
 	}
 
+	if up.Dev.IsHybridModeEnabled() {
+		oktetoLog.Infof("stopping local process...")
+		up.shutdownHybridMode()
+	}
+
 	oktetoLog.Info("completed shutdown sequence")
 	up.ShutdownCompleted <- true
 
 }
 
+func (up *upContext) shutdownHybridMode() {
+	if up.hybridCommand != nil {
+		pid := up.hybridCommand.Process.Pid
+		if existProcess(pid) {
+			if err := killProcess(pid); err != nil {
+				oktetoLog.Warning("failed to stop gracefully local process related to command executed in hybrid mode: %s", err.Error())
+			}
+
+			waitTimeout := 15 * time.Second
+			hasFinished := hasHybridCommandFinishes(pid, waitTimeout)
+			if !hasFinished {
+				oktetoLog.Warning("timeout waiting to finish hybrid command gracefully")
+			}
+		}
+	}
+}
+
+// killProcess sends a SIGTERM signal to the given process
+func killProcess(pid int) error {
+	p, err := os.FindProcess(pid)
+	if err != nil {
+		return err
+	}
+	return p.Signal(syscall.SIGTERM)
+}
+
+func existProcess(pid int) bool {
+	found, err := ps.FindProcess(pid)
+	if err == nil && found == nil {
+		return false
+	}
+	return true
+}
+
+func hasHybridCommandFinishes(pid int, timeout time.Duration) bool {
```

Comment:
```suggestion
func hasHybridCommandFinished(pid int, timeout time.Duration) bool {
```

It sounds like a typo?